siatest/renter/renter_test.go

```diff
@@ -75,3 +80,52 @@ func testUploadDownload(t *testing.T, tg *siatest.TestGroup) {
 		t.Error(err)
 	}
 }
+
+// testDownloadMultipleLargeSectors downloads multiple large files (>5 Sectors)
+// in parallel and makes sure that the downloads are blocking each other.
+func testDownloadMultipleLargeSectors(t *testing.T, tg *siatest.TestGroup) {
+	// parallelDownloads is the number of downloads that are run in parallel.
+	parallelDownloads := 10
+	// fileSize is the size of the downloaded file.
+	fileSize := int(10*modules.SectorSize) + siatest.Fuzz()
+	// set download limits and reset them after test.
+	ratelimit.SetLimits(int64(fileSize)*2, 0, modules.SectorSize)
+	defer ratelimit.SetLimits(0, 0, 0)
```

Comment:
If we do something like this we will be unable to run this test in parallel with other tests in any way.

I guess in general, tests on a TestGroup should not be run in parallel within that same TestGroup, that could cause a lot of issues. Maybe we should make the ratelimits specific to a TestGroup though?

Reply:
Thinking about it a little more...

should ratelimits be specific to each `server`? I think if you are a user running multiple `siad` servers, you may want to ratelimit them independently, even though they may all be running from the same go runtime.

Reply:
Makes sense but this also seems like it would belong into a different PR. That requires changing the ratelimit package itself.